Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -25,17 +25,20 @@
 import java.util.HashMap;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.model.Charity;
+import com.google.model.Tag;
 
-public final class FindTrendingCharities {
+public final class FindTrendingCharities extends DbSetUpUtils {
 
     //datastore set up
-    private DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
-    private DbCalls db = new DbCalls(ds);
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
```

Comment:
you should not need these again here since you have it base class. Directly use it.

Reply:
fixed